common/lib/capa/capa/capa_problem.py

```diff
@@ -215,6 +216,15 @@ def make_xml_compatible(self, tree):
         This translation takes in the new format and synthesizes the old option= attribute
         so all downstream logic works unchanged with the new <option> tag format.
         """
+        # Convert the existing problem's XML to new format
+        # <problem>...</problem> to <problem><question>...</question></problem>
+        questions = tree.xpath('//problem/question')
+        if not questions:
+            tree.tag = 'question'
+            problem = etree.Element('problem')
+            problem.insert(0, tree)
+            tree = problem
+
```

Comment:
I'm not sure this is a good way to handle/phrase things, conceptually. The new format is a more verbose special case. I'm not sure this is fair to treat as a backwards-compatibility. I suspect most people will prefer to author in the original format. That should still be a fully support/recommended way to do things.